src/devices/mailbox.js

```diff
@@ -166,6 +162,8 @@ export function buildMailbox(state) {
     }
   }
 
+  // Functions made available to the host: these are used for inbound
```

Comment:
I'd either move this back up to cover `deliverInbound`, or reword it to "`deliverInbound` is made available to the host, for it to deliver inbound messages..". I don't want to give the impression that `srcPath` and `endowments` are made available to the host (well, they are, but it only uses them at config time, and passes them directly into the controller/kernel to build the device, whereas `deliverInbound` is used at runtime, every time a new message arrives)

Reply:
done